extension/oauth2clientauthextension/config.go

```diff
@@ -35,6 +37,43 @@ type Config struct {
 	// ClientSecretFile is the file pathg to read the application's secret from.
 	ClientSecretFile string `mapstructure:"client_secret_file"`
 
+	// ClientCertificateKeyID is the Key ID to include in the jwt. Only used if
+	// GrantType is set to "urn:ietf:params:oauth:grant-type:jwt-bearer".
+	ClientCertificateKeyID string `mapstructure:"client_certificate_key_id"`
+
+	// ClientCertificateKey is the application's private key. Only used if
+	// GrantType is set to "urn:ietf:params:oauth:grant-type:jwt-bearer".
+	ClientCertificateKey configopaque.String `mapstructure:"client_certificate_key"`
```

Comment:
can all those new parameters be defined in the README.md?

Reply:
yeah for sure, I forgot to update it with an example, sorry